src/commonMain/kotlin/fr/acinq/lightning/Features.kt

```diff
@@ -21,6 +21,9 @@ sealed class Feature {
     abstract val rfcName: String
     abstract val mandatory: Int
     val optional: Int get() = mandatory + 1
+    abstract val init: Boolean
+    abstract val nodeAnnouncement: Boolean
+    abstract val invoice: Boolean
```

Comment:
there is no `channelAnnouncement` flag, is it because we assume that channels will always be private ?

Reply:
It could be a good reason, but it's just because there's simply no channel feature yet in the spec, and it's unclear how it would even be used.